scripts/start.js

```diff
@@ -115,6 +108,16 @@ checkBrowsers(paths.appPath, isInteractive)
             }
             console.log(chalk.cyan('Starting the development server...\n'));
             openBrowser(urls.localUrlForBrowser);
+
+            const connection = require('../src/db/connection');
+            connection.start()
```

Comment:
Abstracted out connection so that it'll call a custom `start` function instead of the built-in `sync` method. This is because the `start` method has logic to checks for all env vars are set before trying to make a connection.